src/server/report/report.service.ts

```diff
@@ -82,6 +83,7 @@ export class ReportService {
         isSEAS: isSEASEnumToString(course.isSEAS),
         isUndergrad: course.isUndergraduate ? 'Undergraduate' : 'Graduate',
         notes: course.notes,
+        sameAs: course.sameAs,
```

Comment:
The course's `sameAs` field value refers to a course's parent (if one exists). If you take a look at the `Course` entity, the `sameAs` field points to one course. From the description of the ticket, I think that we want to also pull in the course's siblings to the excel report if any exist. The logic to get this information is a bit more complicated. 

I think one way to accomplish this would be to create a separate helper function that returns an array of objects. Each key in the object would be a parent course, and the value would be an array of its the parent course's children. From there you could parse this result to get all of the `sameAs` course information. 

There are likely other ways to tackle this problem, but that was just the first idea that came to mind.

Reply:
Sorry for the confusion - you can disregard the above comment. The sameAs courses are showing as expected.